hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java

```diff
@@ -293,6 +293,21 @@ public static <K, V> Map<V, K> getReverseMap(Map<K, List<V>> map) {
             .collect(Collectors.toMap(Pair::getKey, Pair::getValue));
   }
 
+  /***
+   * Removed all files and dirs in the given dir recursively.
+   */
+  public static boolean deleteDirectory(File dir) {
+    File[] allContents = dir.listFiles();
```

Comment:
if its a file instead of dir, wouldn't it throw exception ?

Reply:
No, it reads all the file and dir in path and deletes them recursively.

Reply:
I believe @prashantpogde is asking what happens if `dir` is a file rather than a directory.